test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -328,6 +328,73 @@ BOOST_AUTO_TEST_CASE(public_state_variable)
 	checkNatspec(sourceCode, "test", userDoc, true);
 }
 
+BOOST_AUTO_TEST_CASE(public_state_variable_struct)
+{
+	char const* sourceCode = R"(
+		contract Bank {
+			struct Coin {
+				string obverseGraphicURL;
+				string reverseGraphicURL;
+			}
+
+			/// @notice Get the n-th coin I own
+			/// @return obverseGraphicURL Front pic
```

Comment:
Can you swap this and the next line, so that we can verify that the association is by name and not by position?

Reply:
They are actually associated by name AND position. If either is wrong, it will fail. That was also the behavior of the previous code which only worked for functions though, not state variables. 

I am currently implementing that the order doesn't matter, but something like _only order_ (e.g. no parameter names) feels unnecessary at this point.. let me know if you disagree